Sources/AlgoliaSearch-Client/Query.swift

```diff
@@ -805,6 +805,28 @@ open class Query : AbstractQuery {
         set { self["percentileComputation"] = Query.buildBool(newValue) }
     }
     
+    /// Values applicable to the `sortFacetValuesBy` parameter.
+    public enum SortFacetValuesBy: String {
+        /// Facet values are sorted by decreasing count, the count being the number of records containing this facet value in the results of the query (default behavior).
+        case count = "count"
+        /// Facet values are sorted by increasing alphabetical order.
+        case alpha = "alpha"
+    }
+    
+    /// Controls how the facet values are sorted within each faceted attribute.
+    public var sortFacetValuesBy: SortFacetValuesBy? {
+        get {
+            if let value = self["sortFacetValuesBy"] {
```

Comment:
guard let maybe? both way work, I guess

Reply:
I like guard, more compact and less indentation